docs/src/Tabs.doc.js

```diff
@@ -34,6 +34,12 @@ card(
         type: `({ event: SyntheticMouseEvent<>, activeTabIndex: number }) => void`,
         required: true,
       },
+      {
+        name: 'wrap',
+        type: 'boolean',
+        defaultValue: false,
```

Comment:
we usually only add a `defaultValue` if its explicitly assigned via a react components default props. i would either remove this line here, or add it as a default to the component